ihaskell-display/ihaskell-widgets/src/IHaskell/Display/Widgets/Box/HBox.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE TypeSynonymInstances #-}
+
+{-# OPTIONS_GHC -fno-warn-orphans  #-}
+
+module IHaskell.Display.Widgets.Box.HBox
+  ( -- * The HBox widget
+    HBox
+    -- * Constructor
+  , mkHBox
+  ) where
+
+-- To keep `cabal repl` happy when running from the ihaskell repo
```

Comment:
Can we remove this?

Reply:
Hmm, I see that this same line is present in the codebase prior to this change. I guess it doesn't really do any harm, It's up to you if you want to leave it in or remove it everywhere else too.

Reply:
Should I remove also the `mport Prelude`? Or only the comment?

Reply:
The `import Prelude` as well, but given how it's all over the `ihaskell-widgets` code it doesn't particularly matter.